website/docs/design-patterns/layered-stack-configuration.md

```diff
@@ -0,0 +1,328 @@
+  - catalog/network-firewall/defaults
+  - catalog/waf/defaults
+  # Import other Firewall components
+```
+
+Add the following configuration to the `stacks/layers/networking.yaml` layer manifest:
+
+```yaml title="stacks/layers/networking.yaml"
+import:
+  # Import the related component manifests into this layer manifest
+  - catalog/vpc/defaults
+  # Import other Networking components
+```
+
+Add the following configuration to the `stacks/layers/eks.yaml` layer manifest:
+
+```yaml title="stacks/layers/eks.yaml"
+import:
+  # Import the related component manifests into this layer manifest
+  - catalog/eks/defaults
+```
+
+Import the required layers into the `stacks/orgs/acme/plat/dev/us-east-2.yaml` top-level stack manifest:
+
+```yaml title="stacks/orgs/acme/plat/dev/us-east-2.yaml"
+import:
+  # The `orgs/acme/plat/dev/_defaults` and `mixins/region/us-east-2` manifests 
+  # define the top-level Atmos stack `plat-ue2-dev`
+  - orgs/acme/plat/dev/_defaults
+  - mixins/region/us-east-2
+  # Import the layers (groups of components)
+  - layers/load-balancers
+  - layers/data
+  - layers/dns
+  - layers/logs
+  - layers/notifications
+  - layers/firewalls
+  - layers/networking
+  - layers/eks
+```
+
+Import the required layers into the `stacks/orgs/acme/plat/dev/us-west-2.yaml` top-level stack manifest:
+
+```yaml title="stacks/orgs/acme/plat/dev/us-west-2.yaml"
+import:
+  # The `orgs/acme/plat/dev/_defaults` and `mixins/region/us-west-2` manifests 
+  # define the top-level Atmos stack `plat-uw2-dev`
+  - orgs/acme/plat/dev/_defaults
+  - mixins/region/us-west-2
+  # Import the layers (groups of components)
+  - layers/load-balancers
+  - layers/data
+  - layers/dns
+  - layers/logs
+  - layers/notifications
+  - layers/firewalls
+  - layers/networking
+  - layers/eks
+```
+
+Import the required layers into the `stacks/orgs/acme/plat/prod/us-east-2.yaml` top-level stack manifest:
+
+```yaml title="stacks/orgs/acme/plat/prod/us-east-2.yaml"
+import:
+  # The `orgs/acme/plat/prod/_defaults` and `mixins/region/us-east-2` manifests 
+  # define the top-level Atmos stack `plat-ue2-prod`
+  - orgs/acme/plat/prod/_defaults
+  - mixins/region/us-east-2
+  # Import the layers (groups of components)
+  - layers/load-balancers
+  - layers/data
+  - layers/dns
+  - layers/logs
+  - layers/notifications
+  - layers/firewalls
+  - layers/networking
+  - layers/eks
+```
+
+<br/>
+
+Similarly, import the required layers into the other top-level stacks for the other organizations, OUs/tenants, accounts and regions.
+Make sure to import only the layers that define the component that need to be provisioned in the stacks.
+
+## Benefits
+
+The **Layered Stack Configuration** pattern provides the following benefits:
+
+- Allows to group Atmos components by category or function
+
+- Allows splitting the components into layers. Each layer can be managed and modified independent of the other layers, possibly by different
+  people or teams
```

Comment:
```suggestion
  people or teams. Furthermore, controls like GitHub's [`CODEOWNERS`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) can be leveraged so that specific teams or individuals must review changes to these files.
```